VERSION

```diff
@@ -0,0 +1 @@
+1.1.12
```

Comment:
Do we usually just keep this in the package.json? Just want to avoid updating multiple places to cause out of sync issues.